package/yast2-metapackage-handler.changes

```diff
@@ -1,3 +1,10 @@
+-------------------------------------------------------------------
+Wed Apr  7 19:16:18 UTC 2021 - Dirk Müller <dmueller@suse.com>
+
+- stop packaging directories owned by filesystem
```

Comment:
do you have bugzilla number for this change? or jira entry or something similar? some ID?

Reply:
no, but I can create one if you need it.

Reply:
it would be good. We have that rule to have rationale behind all changes for future, so we can easily revisit if it still makes sense ( well, in that case it is easy, but who knows what future brings )

Reply:
I've added it.